src/documentation/tooling/evaluation.md

```diff
@@ -0,0 +1,292 @@
+## Documentation tooling &mdash; evaluation and implementation
+
+> **Disclaimer**
+>
+> Although we have attempted to make our evaluation as objective as possible,
+> subjective biases and preferences will usually influence choice of tooling
+> (cf. the [editor war](https://en.wikipedia.org/wiki/Editor_war)).
+> Additionally, our evaluation methods are informal to some extent, not strictly 
+> empirical, and non-reproducible. Nevertheless we feel that publishing them
+> here may be useful to record the criteria we have taken into account.
+
+---
+
+Requirements for single tool documentation software (cf. [documentation tooling section](./index.md)):
+- (1) Sustainability
+- (2) Single tool toolchain
+- (3) Usability  
+    - (3a) Human-readable source format  
+    - (3b) Javadoc integration  
+    - (3c) Continuous integration capabilities  
+    - (3d) Maintainability
+    - (3e) Maintainability (dependencies)
+    - (3f) Usability of representations
+- (4) Different representation forms  
+
+---
+
+To evaluate which documentation tool may be the most suitable for our needs, we
+have marked it with a value from 1-5 for each requirement, with 1 being the
+lowest ("worst") mark and 5 the highest ("best").
+
+
+|     Tool     | 1 | 3a | 3b | 3c | 3d | 3e | 3f | x̄(3)  | 4 |
+|--------------|---|----|----|----|----|----|----|-------|---|
+| Sphinx (rST) | 5 | 3  | 1  | 3  | 4  | 3  | 4  |  3.0  | 4 |
+| Sphinx (CM)  | 3 | 5  | 1  | 3  | 2  | 3  | 4  |  3.0  | 4 |
+| Asciidoctor  | 3 | 4  | 1  | 4  | 3  | 3  | 3  |  3.0  | 3 |
+| mkDocs       | 3 | 5  | 1  | 3  | 4  | 3  | 2  |  3.0  | 1 |
+| mdBook       | 4 | 5  | 1  | 3  | 4  | 4  | 5  |  3.67 | 3 |
+| Jekyll       | 4 | 5  | 1  | 3  | 4  | 2  | 2  |  2.83 | 2 |
+
+
+### Sustainability
+
+As of now, reliable measures for predicting the sustainability of a software do 
+not exist [1], and intrinsically, the actual sustainability of a software can
+only be determined in hindsight. Therefore, assessing the sustainability
+potential for a software is a qualitative process, partly driven by the
+requirements of the project for which the software is assessed.
+
+Despite this constraint, assumptions over the sustainability potential of a
+documentation software project can be based on some assessable factors, e.g., 
+age of the software, approximated user base, development status, frequency of 
+contributions, architecture, pervasiveness of a technological community,
+level of documentation, maturity of its dependencies, etc.
+
+In the evaluation process, we have tried to approximate each candidate's
+potential for sustainability, which we present in the following.
+Additionally, we give the [SourceRank metric for high quality packages]() used by libraries.io, based on the actual products we would employ.
```

Comment:
Where is the link for SourceRank?

Reply:
It's in 3b52b64138eab83190dd974f737dde6e4b5dcc13.